docs/content/concepts/assets/multi-assets.mdx

```diff
@@ -132,3 +134,23 @@ def my_complex_assets(a, b):
     # d only depends on b
     yield Output(value=b + 1, output_name="d")
 ```
+
+---
+
+## Multi-asset code versions
```

Comment:
I moved the multi-asset specific code version docs to the multi-asset concept page because I felt that they didn't fit in the assets concept page. The asset concept page is probably one of the first ones a user looks at after the tutorial, and this particular section introduces two new concepts (multi assets and ops) that are not strictly necessary yet